src/main/java/org/jboss/xavier/analytics/pojo/output/AnalysisModel.java

```diff
@@ -51,6 +53,9 @@
     private Date lastUpdate;
     private String owner;
 
+    @Column(length = 512)
```

Comment:
yes, but if I downsize it , it will crash because we already have values with that size
in anycase it's a VARCHAR so it doesnt harm at all

Reply:
Delete rows in the CI DB that prevents you from changing this please since there's no need to keep WIP solutions.
And hence remove the annotation so that we can go with the default values as we do for any other String.

Reply:
ok